app/src/lib/git/for-each-ref.ts

```diff
@@ -102,6 +102,14 @@ export async function getBranches(
       continue
     }
 
+    if (ref.startsWith('refs/remotes/github-desktop-')) {
```

Comment:
There's a constant we export for `github-desktop`, maybe use that?
https://github.com/desktop/desktop/blob/71fb470e4fcc234a4d471b330704ec71432f44d8/app/src/lib/stores/pull-request-store.ts#L26